cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -144,10 +145,12 @@ fn insert_cmp(
     let rhs = builder.use_var(rhs);
 
     let res = if opcode == Opcode::Fcmp {
+        let cc = *fgen.u.choose(FloatCC::all())?;
+
         // Some FloatCC's are not implemented on AArch64, see:
         // https://github.com/bytecodealliance/wasmtime/issues/4850
-        let float_cc = if cfg!(target_arch = "aarch64") {
-            &[
+        if matches!(fgen.target_triple.architecture, Architecture::Aarch64(_))
```

Comment:
Let's add a comment here like the one in `generate_instructions`:
```suggestion
        // We filter out condition codes that aren't supported by the target at
        // this point after randomly choosing one, instead of randomly choosing a
        // supported one, to avoid invalidating the corpus when these get implemented.
        if matches!(fgen.target_triple.architecture, Architecture::Aarch64(_))
```